### PR TITLE
GDPAPPS-400: implemented presentation framework

### PR DESCRIPTION
**Pull request by mhockenbury**
Updated requirements.txt and setup.py for framework
Implemented presentation framework for Couchbase calls to CORE couchbase

Notes: the updates to the configuration JSON were required to match up to the expectations of the framework since we pass the config object to it to instanitate a CB connection. So the config JSON here is a little messy for now, will need to clean this up a bit, also may be worthwhile to match up our naming conventions for akira's CB stuff with framework (eg. multiget vs get_multi)

~~TODO: still need to dynamically get different asset formats (currently hardcoded to 'full')~~

### CODE REVIEW

File: appserv_akira/application.py [hunk 1]
```diff
@@ -29,14 +29,14 @@
 scheduler.start()
 
 
-def get_user_feed(userId, field_list=None):
+def get_user_feed(userId, field_list=None, transform='full'):
```
Comment (alon85) [inline appserv_akira/application.py]: write a TODO to change this to default to assetLite and make configurable

**Comment by alon85** [comment]
Take out "_asset" since it is taken care of automatically in framework.

**Comment by alon85** [comment]
You're missing a coma in your setup.py after "APScheduler"

**Comment by mhockenbury** [comment]
@alon85 I took out the "asset_" prefix I was adding, fixed the comma, and implemented loading configs by setting the `ENVIRONMENT`env var to the appropriate value (ex. local, development)

**Pull request merged**